ecosystem/resources/members/QCRAFT-AutoSchedulQ.toml

```diff
@@ -0,0 +1,13 @@
+name = "QCRAFT AutoScheduler"
+url = "https://github.com/Qcraft-UEx/QCRAFT-AutoSchedulQ"
+description = "AutoScheduler is a library that allows developers to optimize the scheduling and execution of quantum circuits on the IBM Quantum platform by reducing waiting times and costs through the automatic combination of multiple executions of the same circuit in a single task. This tool allows the composition of the same quantum circuit to be executed on the same task, thus taking advantage of the maximum number of qubits available in the simulator or quantum computer. This tool allows maximizing the use of available qubits and reducing the total number of shots required for the execution of the same circuit without a significant impact on the results. As an example and to facilitate the understanding of the AutoScheduler library, if you wanted to run 1000 shots of a quantum circuit requiring 30 qubits on any IBM machine with 127 qubits, this library would compose a task with 4 times the circuit using 120 qubits and reducing the number of shots from 1000 to 250."
```

Comment:
This needs to be <135 characters. Can you rephrase please? Look at other descriptions on https://www.ibm.com/quantum/ecosystem to get a feel for the style.

Reply:
I edited the original issue, it's that ok? Or it is better to create a new issue?